### PR TITLE
WIP: [CFM-1034] -Inviter can customize title and body

### PR DESCRIPTION
**Pull request by ybaras**
https://issuetracker.amplexor.com/jira/browse/CFM-1034

### CODE REVIEW

**Comment by ybaras** [comment]
@ordavidil IDK why CS fails. Line 524 is exactly 50 characters long :-1: .

**Comment by ybaras** [comment]
@ordavidil The email templates will be handled in another PR for [CFM-1029|https://issuetracker.amplexor.com/jira/browse/CFM-1029].

**Comment by ordavidil** [comment]
@ybaras 
are you sure this PR should've compared with `develop` ?  cause i'm pretty sure i've seen some the code in here on a different PR

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_invite/c4m_og_invite.module [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  $group_request_membership_value = c4m_og_get_group_membership_request($group);
+
+  return $group_request_membership_value == 'open' && og_user_access($group_type, $gid, 'member');
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function c4m_og_invite_form__og_invite_people_new_users_form_alter(&$form, &$form_state, $form_id) {
+  $form['og_invite_people']['tamessage'] = array(
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_invite/c4m_og_invite.module]: be explicit with the field name

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_invite/c4m_og_invite.module [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  );
+}
+
+/**
+ * Implements hook_mail_alter().
+ */
+function c4m_og_invite_mail_alter(&$message) {
+  if ($message['key'] == 'register_admin_created') {
+    // This is the message sent by the og_invite_people module.
+    // @TODO: Add the correct tpl.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_invite/c4m_og_invite.module]: Remove the `TODO`

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_invite/c4m_og_invite.module [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function c4m_og_invite_form__og_invite_people_new_users_form_alter(&$form, &$form_state, $form_id) {
+  $form['og_invite_people']['tamessage'] = array(
+    '#type' => 'textarea',
+    '#title' => 'Add a message',
+    '#description' => 'Add a message to the invitation (optional)',
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_invite/c4m_og_invite.module]: use `t()`

**Pull request closed without merging**